### PR TITLE
HOSTEDCP-1511: enable and document kubevirt custom default network

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
After merging https://github.com/openshift/hypershift/pull/3528, the `hcp` command tool is missing the `--attach-default-network` option to disable it and also the project is missing the documentation about attaching just to a custom network as primary.

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: docs/content/how-to/kubevirt/configuring-network.md [hunk 1]
```diff
@@ -32,3 +32,29 @@ hcp create cluster kubevirt \
 In this example, the KubeVirt VMs will have interfaces attached to the networks
 for the NetworkAttachmentDefinitions network1 and network2 which reside in
 namespace my-namespace.
+
+## Using managed networks as default
```
Comment (davidvossel) [inline docs/content/how-to/kubevirt/configuring-network.md:36]: I'm not sure the term `managed networks` will be interpreted accurately enough.

I think what we want to communicate is someone can disable usage of the pod network and use a secondary network (multus) as the default network for the worker node.

So maybe `Using Secondary Network as Default`?

File: docs/content/how-to/kubevirt/configuring-network.md [hunk 2]
```diff
@@ -32,3 +32,29 @@ hcp create cluster kubevirt \
...
+
+Users managing a network (DHCP, routing, etc...) can use that network 
+as the default one for the kubevirt hosted clusters, to do so
+disable pod default network and attach an additional one that connects to it 
+with the hcp command line tool arguments `--attach-default-network=false` and 
+`--additional-network`.
```
Comment (davidvossel) [inline docs/content/how-to/kubevirt/configuring-network.md:42]: what happens if someone sets `-attach-default-network=false` then adds multiple addtional networks? Which one will be the default?

Comment (qinqon) [reply docs/content/how-to/kubevirt/configuring-network.md:42]: If have just remeber that we activated nodeip-configuration at MCO (https://github.com/openshift/machine-config-operator/pull/4039), with that we can do what is stated at https://docs.openshift.com/container-platform/4.15/support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues

**Review by davidvossel — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**